isotovideo

```diff
@@ -89,13 +133,27 @@ sub _get_version_string () {
     return "Current version is $thisversion [interface v$OpenQA::Isotovideo::Interface::version]";
 }
 
+sub _exit_code_from_test_results () {
+    my @results = glob(path(bmwqemu::result_dir(), "/result-*.json"));
+    return EXIT_STATUS_ERR_NO_TESTS if scalar @results == 0;
+    my $did_fail = 0;
+    for my $result_file_path (@results) {
+        my $result_file = path(Mojo::Path->new($result_file_path)->canonicalize);
```

Comment:
I don't believe you really want to use `Mojo::Path` here. That's for canonicalizing HTTP paths, not file system.

Reply:
Maybe `path($result_file_path)->realpath`?

Reply:
I wanted something to remove duplicated `/` on paths, but in latest commit that doesn't happen anymore.